src/Compiler/Checking/SignatureConformance.fs

```diff
@@ -307,8 +307,11 @@ type Checker(g, amap, denv, remapInfo: SignatureRepackageInfo, checkingSig) =
                               if sigHasInlineIfLambda && not implHasInlineIfLambda then 
                                   errorR(Error (FSComp.SR.implMissingInlineIfLambda(), m))
 
-                              implArgInfo.Name <- sigArgInfo.Name
-                              implArgInfo.Attribs <- attribs))) && 
+                              implArgInfo.OtherRange <- sigArgInfo.Name |> Option.map (fun ident -> ident.idRange)
+                              sigArgInfo.OtherRange <- implArgInfo.Name |> Option.map (fun ident -> ident.idRange)
+
+                              implArgInfo.Name <- implArgInfo.Name |> Option.orElse sigArgInfo.Name
```

Comment:
So we actually had the signature range info before already, because the `Name` is an `Ident` which contains the range and it was set here to the signature's value. But possibly the range from the `Name` is never used. Anyway it seems wrong so I changed it to only take the signature value if we currently have none. But maybe I'd go even further and in this case just take the identifier name + `range.Zero`.